lib/matplotlib/axes/_axes.py

```diff
@@ -5227,88 +5227,113 @@ def pcolor(self, *args, **kwargs):
         """
         Create a pseudocolor plot of a 2-D array.
 
-        .. note::
-
-            pcolor can be very slow for large arrays; consider
-            using the similar but much faster
-            :func:`~matplotlib.pyplot.pcolormesh` instead.
-
         Call signatures::
-
           pcolor(C, **kwargs)
           pcolor(X, Y, C, **kwargs)
 
-        *C* is the array of color values.
-
-        *X* and *Y*, if given, specify the (*x*, *y*) coordinates of
-        the colored quadrilaterals; the quadrilateral for C[i,j] has
-        corners at::
-
-          (X[i,   j],   Y[i,   j]),
-          (X[i,   j+1], Y[i,   j+1]),
-          (X[i+1, j],   Y[i+1, j]),
-          (X[i+1, j+1], Y[i+1, j+1]).
-
-        Ideally the dimensions of *X* and *Y* should be one greater
-        than those of *C*; if the dimensions are the same, then the
-        last row and column of *C* will be ignored.
-
-        Note that the column index corresponds to the
-        *x*-coordinate, and the row index corresponds to *y*; for
-        details, see the :ref:`Grid Orientation
-        <axes-pcolor-grid-orientation>` section below.
+        pcolor can be very slow for large arrays; consider
+          using the similar but much faster
```

Comment:
The problem is that this line is indented and it confuses sphinx; aligning it with the previous line should fix it.